server/src/main/scala/actors/MinuteLookups.scala

```diff
@@ -34,22 +37,30 @@ trait MinuteLookupsLike {
     requestAndTerminateActor.ask(RequestAndTerminate(actor, container)).mapTo[MinutesContainer[StaffMinute, TM]]
   }
 
-  val primaryQueuesLookup: (Terminal, SDateLike) => Future[Option[MinutesContainer[CrunchMinute, TQM]]] = (terminal: Terminal, date: SDateLike) => {
-    val actor = system.actorOf(TerminalDayQueuesActor.props(terminal, date, now))
+  val primaryQueuesLookup: MinutesLookup[CrunchMinute, TQM] = (terminal: Terminal, date: SDateLike, maybePit: Option[MillisSinceEpoch]) => {
+    val props = maybePit match {
+      case None => TerminalDayQueuesActor.props(terminal, date, now)
+      case Some(pointInTime) => TerminalDayQueuesActor.propsPointInTime(terminal, date, now, pointInTime)
+    }
+    val actor = system.actorOf(props)
     requestAndTerminateActor.ask(RequestAndTerminate(actor, GetState)).mapTo[Option[MinutesContainer[CrunchMinute, TQM]]]
   }
 
-  val secondaryQueuesLookup: (Terminal, SDateLike) => Future[Option[MinutesContainer[CrunchMinute, TQM]]] = (terminal: Terminal, date: SDateLike) => {
+  val secondaryQueuesLookup: MinutesLookup[CrunchMinute, TQM] = (terminal: Terminal, date: SDateLike, _: Option[MillisSinceEpoch]) => {
```

Comment:
should we be consistent about naming things as lookup vs legacy lookup

Reply:
I've renamed all secondary to legacy, and removed the primary pre/postfix